data-migration: add dmctl command mode (#2097)
pingcap/docs: reference/tools/data-migration/manage-tasks.md

This document describes how to manage and maintain the data replication task using the [dmctl](/reference/tools/data-migration/overview.md#dmctl) component. For the Data Migration cluster deployed using DM-Ansible, the dmctl binary file is in `dm-ansible/dmctl`.

> The dmctl support the interactive mode for manual operation, and support the command mode for the script.

Reviewer: ```suggestion
The dmctl component supports the interactive mode for manual operations, and also supports the command mode for the script.
```

## dmctl interactive mode

> This section described the basic use of dmctl commands in interactive mode.

Reviewer: ```suggestion
This section described the basic use of dmctl commands in the interactive mode.
```

> This section described the basic use of dmctl commands in the interactive mode.

Reviewer 2: @toutdesuite Why use the past tense?

Author: @lilin90 The mistake is resolved in https://github.com/pingcap/docs/pull/2118. Thanks for pointing it out.

### Task management overview

> Enters the interactive mode to interact with DM-master.

Reviewer 2: `Enters` is weird. This is not a code comment.

Author: @lilin90 The mistake is resolved in #2118. Thanks for pointing it out.

>
> Normally, you do not need to use this command. Use it only when you are sure that the `task => DM-workers` mapping exists, but you are still prompted to refresh while you are executing other commands.

## dmctl command mode

> The command mode differs from the interactive mode in that you need to perform the task operation right after executing the dmctl command. The parameters of the task operation in command mode is the same as that in interactive mode.

Reviewer: @toutdesuite Please check with the author whether `命令模式跟交互模式的区别是，执行命令时只需要在 dmctl 命令后紧接着执行任务操作，任务操作同交互模式的参数一致。` means the edited version, thanks!
```suggestion
The command mode differs from the interactive mode in that you need to append the task operation right after the dmctl command. The parameters of the task operation in the command mode are the same as those in the interactive mode.
```

Author: @TomShawn The author confirmed that the translation is ok. FYI

> **Note:**
>

> > + A dmctl command follows a task command only.

Reviewer: ```suggestion
> + A dmctl command must be followed by only one task operation.
```

> > + The task operation can only be placed at the end of the dmctl command.

Reviewer: ```suggestion
> + The task operation can be placed only at the end of the dmctl command.
```
